src/spawn.rs

```diff
@@ -0,0 +1,78 @@
+//! Entity spawning utilities.
+
+// Unused utilities may trigger these lints undesirably.
+#![allow(dead_code)]
+
+use bevy::ecs::system::EntityCommand;
+use bevy::ecs::system::EntityCommands;
+use bevy::prelude::*;
+
+/// Re-exported extension traits.
+#[allow(unused_imports)]
+pub mod prelude {
+    pub use super::{AddExt as _, SpawnExt as _, WorldSpawnExt as _};
+}
+
+/// An extension trait that provides helper functions for deferred entity spawning.
+pub trait SpawnExt {
+    // Workaround for https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086.
```

Comment:
```suggestion
    // Workaround for <https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086>.
```
Link does not point to the comment due to the period